README.md

```diff
@@ -1,3 +1,5 @@
+<img align="right" width="150" height="150" src="https://avatars0.githubusercontent.com/u/6076200">
```

Comment:
1. Plain HTML in markdown just to get alignment/size? Is it really worth it?
2. We don't even have a fancy user icon

Reply:
What does the icon mean? 

Do you know it's a Chinese character  `早`？

Reply:
Logo is a better term. Right now we use this image of 早. We used 早 because of the idea of speed and because it's present in one of the common spellings of Chihaya (千早). But Chihaya isn't necessarily the fastest tracker nor intends to be. It's mostly about being a flexible, clean foundation for people building systems on top of BitTorrent.